src/contexts/CgiTokenMarketData/CgiTokenMarketDataContext.ts

```diff
@@ -2,6 +2,7 @@ import { createContext } from 'react'
 
 interface CgiTokenMarketDataValues {
   prices?: number[][]
+  dayPrices?: number[][]
```

Comment:
how do `dayPrices` differ from `prices`?
If we add support for 3M and 1Y prices in the future will there be `threeMonthPrices` and `oneYearPrices`?

Reply:
prices is everything that CG will give us, in daily intervals. dayPrices is hourly for the most recent 48 hours. 

I'll gladly take better name suggestions, especially if the dayPrices is turning into a week or more to match current granularity. Will await your suggestions

Reply:
I see. yeah maybe `oneWeekPrices` or `oneMonthPrices` depending on which you go with.